docs/Grambank_most_updated_sheet.tsv

```diff
@@ -188,15 +188,15 @@ GB519	Can mood be marked by a non-inflecting word (‘auxiliary particle')?	1: p
+GB024a	Is the order of the numeral and noun Num-N?	1: present, 0: absent						"GB024 focuses on the order of cardinal numerals (not ordinal numerals) and nouns in quantified noun phrases. This feature has also been split into two binary features, ""GB024a Is the order of the numeral and noun Num-N?"" and ""GB024b Is the order of the numeral and noun N-Num?"". It is possible to answer 1 or 0 for one of the binarised features, and ""?"" for the other.  In some languages only one order (Numeral-Noun or Noun-Numeral) is possible. In other languages, both orders may be possible. This can take the form of free variation (where Numeral-Noun and Noun-Numeral order can occur interchangeably). The order may also depend on some other grammatical category which is neutral with respect to pragmaticality (e.g. definiteness). If the order depends on the noun, consider only the orders that apply to an open class of nouns and disregard orders that are restricted to a specific closed class of nouns."	numeral, NP	Hannah J. Haynie
+GB024b	Is the order of the numeral and noun N-Num?	1: present, 0: absent						"GB024 focuses on the order of cardinal numerals (not ordinal numerals) and nouns in quantified noun phrases. This feature has also been split into two binary features, ""GB024a Is the order of the numeral and noun Num-N?"" and ""GB024b Is the order of the numeral and noun N-Num?"". It is possible to answer 1 or 0 for one of the binarised features, and ""?"" for the other.  In some languages only one order (Numeral-Noun or Noun-Numeral) is possible. In other languages, both orders may be possible. This can take the form of free variation (where Numeral-Noun and Noun-Numeral order can occur interchangeably). The order may also depend on some other grammatical category which is neutral with respect to pragmaticality (e.g. definiteness). If the order depends on the noun, consider only the orders that apply to an open class of nouns and disregard orders that are restricted to a specific closed class of nouns."	numeral, NP	Hannah J. Haynie
+GB025a	Is the order of the adnominal demonstrative and noun Dem-N?	1: present, 0: absent						"This feature is concerned with the order of the noun and adnominal demonstratives (i.e. demonstratives that modify the noun). There are three possible values to code: 1, before the noun, 2, after the noun, or 3, both orders are possible. Like all questions concerning the order of elements, only the pragmatically unmarked orders are targeted.  This feature has also been split into two binary features, ""GB025a Is the order of the adnominal demonstrative and noun Dem-N?"" and ""GB025b Is the order of the adnominal demonstrative and noun N-Dem?"". It is possible to answer 1 or 0 for one of the binarised features, and ""?"" for the other."	numeral, NP	Jay Latarche and Jeremy Collins

```

Comment:
I messed up and somehow missed GB025. I've fixed it now. I'm sorryl

Reply:
I regenerated the thing with the updated wiki.